src/components/ReportActionItem/IOUAction.js

```diff
@@ -53,7 +57,11 @@ const IOUAction = (props) => {
                         chatReportID={props.chatReportID}
                         onPayButtonPressed={launchDetailsModal}
                         onPreviewPressed={launchDetailsModal}
-                        containerStyles={[styles.cursorPointer]}
+                        containerStyles={[styles.cursorPointer,
+                            props.isFocused
+                                ? styles.iouPreviewBoxHover
+                                : undefined]}
```

Comment:
```suggestion
                        containerStyles={[
                        	styles.cursorPointer,
                            	props.isFocused && styles.iouPreviewBoxHover
                            ]}
```

Reply:
I had to keep the original ternary (it's used in a few other files) because I kept getting console errors as react wouldn't accept the false boolean as a style, even if in an array or object